<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutPartitionAllLeft">
     <item>
      <layout class="QHBoxLayout" name="horizontalPartitionTopButtonsLeft">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutPartitionTopPushButtonsLeft">
         <item>
          <widget class="QPushButton" name="pushButtonUpdatePartitionLeft">
           <property name="maximumSize">
            <size>
             <width>85</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Waree</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="toolTip">
            <string>Update the list of Partitions based on your curently entered credentials. </string>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_34">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="PartitionListWidgetLeft">
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutPartitionBottomButtonsLeft">
       <item>
        <widget class="QPushButton" name="pushButtonPartitionDecommissionLeft">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Disable/Delete the currently selected partition(s).</string>
         </property>
         <property name="text">
          <string>Decommission</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPartitionBackupLeft">
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Backup the selected partition definitions to file. Each partition definition will be written to it's own JSON file. </string>
         </property>
         <property name="text">
          <string>Backup</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPartitionRestoreLeft">
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Restore one or more partition definitions from file. These must be properly formatted JSON, either from a previous backup or created in accordance with the Sumo Logic API documentation. </string>
         </property>
         <property name="text">
          <string>Restore</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPartitionJSONLeft">
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>View Partition(s) as JSON</string>
         </property>
         <property name="text">
          <string>JSON</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_40">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutPartitionCenterButton">
     <item>
      <spacer name="verticalSpacer_19">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>80</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPartitionCopyLeftToRight">
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Copy the selected partition definition from the left org to the right org.</string>
       </property>
       <property name="text">
        <string>Copy -&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPartitionCopyRightToLeft">
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Copy the selected partition definition  from the right org to the left org.</string>
       </property>
       <property name="text">
        <string>&lt;- Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_20">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPartitionFindReplaceCopyLeftToRight">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Coming Soon!</string>
       </property>
       <property name="text">
        <string>Find 
Replace-&gt;
Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPartitionFindReplaceCopyRightToLeft">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Coming Soon!</string>
       </property>
       <property name="text">
        <string>Find 
&lt;-Replace
Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_21">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutPartitionAllRight">
     <item>
      <layout class="QHBoxLayout" name="horizontalPartitionTopButtonsRight">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutPartitionTopPushButtonsRight">
         <item>
          <widget class="QPushButton" name="pushButtonUpdatePartitionRight">
           <property name="maximumSize">
            <size>
             <width>85</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Waree</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="toolTip">
            <string>Update the list of Partitions based on your curently entered credentials. </string>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_37">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="PartitionListWidgetRight">
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutPartitionBottomButtonsRight">
       <item>
        <widget class="QPushButton" name="pushButtonPartitionDecommissionRight">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Disable/Delete the currently selected partition(s).</string>
         </property>
         <property name="text">
          <string>Decommission</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPartitionBackupRight">
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Backup the selected partition(s) to file. Each Partition will be written to it's own JSON file. </string>
         </property>
         <property name="text">
          <string>Backup</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPartitionRestoreRight">
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Restore one or more partition definitions from file. These must be properly formatted JSON, either from a previous backup or created in accordance with the Sumo Logic API documentation. </string>
         </property>
         <property name="text">
          <string>Restore</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonPartitionJSONRight">
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>View Partition(s) as JSON</string>
         </property>
         <property name="text">
          <string>JSON</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_39">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
